### PR TITLE
Refetch contacts while payment

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

File: Token/Models/ContactsManager.h [hunk 1]
```diff
@@ -39,4 +39,6 @@
 
 - (nullable UIImage *)imageForPhoneIdentifier:(nullable NSString *)phoneNumber;
 
+- (void)refreshContacts;
```
Comment (Elland) [inline Token/Models/ContactsManager.h]: This seems redundant to me. Shouldn't `tokenContacts` still be a computed property?

Comment (yuliaveres) [inline Token/Models/ContactsManager.h]: not redundant, definitely not, we access them a lot, while looking for a contact f.e.

Comment (yuliaveres) [inline Token/Models/ContactsManager.h]: we should refetch only when needed

Comment (Elland) [inline Token/Models/ContactsManager.h]: So we're caching results until we know we need to update it? 🤔 Good call. But then shouldn't this be called in more places or just for payments?

Comment (yuliaveres) [inline Token/Models/ContactsManager.h]: So in ither cases I listen for db changes, like adding / removing contacts works perfect

Comment (yuliaveres) [inline Token/Models/ContactsManager.h]: but i am not sure for what changes can i listen in case of payment

Comment (yuliaveres) [inline Token/Models/ContactsManager.h]: it would be the best

Comment (yuliaveres) [inline Token/Models/ContactsManager.h]: ok let me try 🤞

Comment (Elland) [inline Token/Models/ContactsManager.h]: After a lot of digging and experimenting, this looks like the best solution for now.

**Pull request merged**